packages/shared/scope-hoisting/src/hoist.js

```diff
@@ -271,7 +271,10 @@ const VISITOR: Visitor<MutableAsset> = {
       path.replaceWith(t.identifier('null'));
     }
 
-    if (t.matchesPattern(path.node, 'module.bundle')) {
+    if (
+      t.matchesPattern(path.node, 'module.bundle.root') ||
```

Comment:
Is there some precedent (in Webpack?) for `module.bundle.root`?

Reply:
AFAIK module.bundle in general is a parcel thing... I chose root here because it represents the root bundle on the page - the current parcelRequire global. Open to other names.